src/main/resources/avro/biodatamethods.avdl

```diff
@@ -0,0 +1,147 @@
+@namespace("org.ga4gh.methods")
+
+protocol BiodataMethods {
+
+import idl "metadata.avdl";
+import idl "common.avdl";
+import idl "methods.avdl";
+import idl "biodata.avdl";
+
+/******************  /individuals/search  *********************/
+/**
+This request maps to the body of `POST /biosamples/search` as JSON.
+*/
+record SearchIndividualsRequest {
+  /**
+  The dataset to search within.
+  */
+  string datasetId;
```

Comment:
Nope, the purpose of dataset is to provider a bag for ownership
of objects, not grouping for analysis.  So it would not be used
to identify the individuals one did GWAS on.  Its more akin to
an NCBI BioProject.  Google uses it for billing and access
control.

A dataset can reference objects in a another dataset.  It's not
a wall.

Although the GWAS is an interesting use cases for the API.

@calbach dataset is still not clear to many people (I am not
even sure I have it right); could someone work on improving the
documentation?

Daniel Zerbino <notifications@github.com> writes:
> I don't understand the logic for deciding that an object belongs to only one
> dataset... If, say, you wish to specify which individuals you did your GWAS on,
> what object would you use? A dataset no? And we definitely would want
> individuals to be re-used in multiple cohorts.